monai/networks/layers/simplelayers.py

```diff
@@ -163,18 +164,28 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x.reshape(shape)
 
 
-def separable_filtering(x: torch.Tensor, kernels: Union[Sequence[torch.Tensor], torch.Tensor]) -> torch.Tensor:
+def separable_filtering(
+    x: torch.Tensor, kernels: Union[Sequence[torch.Tensor], torch.Tensor], mode: str = "zeros"
+) -> torch.Tensor:
     """
     Apply 1-D convolutions along each spatial dimension of `x`.
 
     Args:
         x: the input image. must have shape (batch, channels, H[, W, ...]).
         kernels: kernel along each spatial dimension.
             could be a single kernel (duplicated for all dimension), or `spatial_dims` number of kernels.
+        mode (string, optional): padding mode passed to convolution class. ``'zeros'``, ``'reflect'``, ``'replicate'``
+            or ``'circular'``. Default: ``'zeros'``. Modes other than ``'zeros'`` require PyTorch version >= 1.5.1. See
+            torch.nn.Conv1d() for more information.
 
     Raises:
         TypeError: When ``x`` is not a ``torch.Tensor``.
     """
+
+    pytorch_version_pre_1_5_1 = tuple(int(x[0]) for x in torch.__version__.split(".")[:3]) < (1, 5, 1)
```

Comment:
On second thought, I think it would be simpler to only support 1.6+ (rather than 1.5.1+), so that I can use the PyTorch version checking built in to MONAI. That would solve this problem and a few of the others you have highlighted.

Reply:
thanks! I guess we could drop torch 1.5 support at the project level soon...

Reply:
As described below, this version check is no longer needed as the new functional implementation supports 1.5.0. These lines have been removed in the latest push.